backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/runMigrations.kt

```diff
@@ -0,0 +1,90 @@
+package app.ehrenamtskarte.backend.migration
+
+import app.ehrenamtskarte.backend.migration.database.MigrationEntity
+import app.ehrenamtskarte.backend.migration.database.Migrations
+import app.ehrenamtskarte.backend.migration.migrations.V1_RemoveEmailIndex
+import java.time.Clock
+import java.time.Instant.now
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.sql.Database
+import org.jetbrains.exposed.sql.SchemaUtils.create
+import org.jetbrains.exposed.sql.exists
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.jetbrains.exposed.exceptions.ExposedSQLException
+import org.slf4j.LoggerFactory
+import org.postgresql.util.PSQLException
+
+private val logger = LoggerFactory.getLogger(Migrations::class.java)
+
+fun runAllMigrations(database: Database, clock: Clock) {
+    runMigrations(database, listOf(V1_RemoveEmailIndex()), clock)
+}
+
+fun runMigrations(database: Database, migrations: List<Migration>, clock: Clock) {
```

Comment:
Can we wrap everything here inside a transaction? That way we would not do anything to the DB (I hope) and can roll back to the last backend code.